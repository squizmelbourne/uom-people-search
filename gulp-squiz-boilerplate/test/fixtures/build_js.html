<!-- build:js test/.tmp/build_js.js -->
    <script src="test/fixtures/js/test.js"></script>
<!-- endbuild -->