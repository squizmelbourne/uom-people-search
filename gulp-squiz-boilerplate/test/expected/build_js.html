<script src="test/.tmp/build_js.js"></script>